chalice/deploy/deployer.py

```diff
@@ -281,7 +282,8 @@ def create_build_stage(osutils, ui, swagger_gen):
             ),
             SwaggerBuilder(
                 swagger_generator=swagger_gen,
-            )
+            ),
+            LambdaEventSourcePolicyInjector(),
```

Comment:
Is this added to the managed policy even if they're not using SQS events?

Reply:
No, the base class does dynamic dispatch based on the class type, so [handle_sqseventsource](https://github.com/aws/chalice/pull/886/files/24ae5b0d0dcd0be19fd69fdff8c5fb2fef12e288#diff-a4c0c52aa7a3b2d98c797c244f0f0e8fR766) is only invoked if that resource is part of your app graph.